### PR TITLE
docs(appium): add axe DevTools for Mobile drivers to Appium other drivers list: Supports accessibility testing for mobile apps

### PR DESCRIPTION
**Pull request by karnett**
## Proposed changes

Adding the axe DevTools for Mobile drivers to the other driver list on the Appium Docs. We've extended the publicly available UiAutomator2 and XCUI drivers to include accessibility testing for anyone wishing to easily catch accessibility issues with their automated tests.

For insight into our testing platform please checkout the [setup documentation](https://docs.deque.com/devtools-mobile/2024.9.18/en/appium-setup). There's also a free trial for those interested in giving it a try.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [Contributing Guide](https://appium.io/docs/en/latest/contributing/)
- [x] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

N/A

### CODE REVIEW

File: packages/appium/docs/en/ecosystem/drivers.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ These drivers are not maintained by the Appium team and can be used to target ad
 |[Tizen](https://github.com/Samsung/appium-tizen-driver)|`--source=npm appium-tizen-driver`|Android|Native|Community / Samsung|
 |[TizenTV](https://github.com/headspinio/appium-tizen-tv-driver)|`--source=npm appium-tizen-tv-driver`|Samsung TV|Web|HeadSpin|
 |[Youi](https://github.com/YOU-i-Labs/appium-youiengine-driver)|`--source=npm appium-youiengine-driver`|iOS, Android, macOS, Linux, tvOS|Native|Community / You.i|
+|[axe UiAutomator 2](https://docs.deque.com/devtools-mobile/appium-setup)|`--source=npm @axe-devtools/axe-appium-uiautomator2-driver`|Android|Native|[Deque](https://www.deque.com)|
+|[axe XCUI Test](https://docs.deque.com/devtools-mobile/appium-setup)|`--source=npm @axe-devtools/axe-appium-xcuitest-driver`|iOS|[Deque](https://www.deque.com)|
```
Comment (eglitise) [inline packages/appium/docs/en/ecosystem/drivers.md:48]: I believe the `Native` value is missing on this line, but you've added it in the previous line

**Review by eglitise — COMMENTED** [review]
Thank you!
Could you please also sign the CLA?

File: packages/appium/docs/en/ecosystem/drivers.md (hunk 1 above)
Comment (KazuCocoa) [inline packages/appium/docs/en/ecosystem/drivers.md:48]:
```suggestion
|[axe UiAutomator2](https://docs.deque.com/devtools-mobile/appium-setup)|`--source=npm @axe-devtools/axe-appium-uiautomator2-driver`|Android|Native|[Deque](https://www.deque.com)|
|[axe XCUITest](https://docs.deque.com/devtools-mobile/appium-setup)|`--source=npm @axe-devtools/axe-appium-xcuitest-driver`|iOS|Native|[Deque](https://www.deque.com)|
```
?

Comment (karnett) [reply packages/appium/docs/en/ecosystem/drivers.md:48]: Ope! Thanks! Working on getting the CLA reviewed by my team now. Thanks! Will update this too.

**Review by KazuCocoa — APPROVED** [review]
lg once CLA has been signed.

**Review by TDL-EdgarsEglitis — APPROVED** [review]

**Comment by karnett** [comment]
@KazuCocoa updated, signed and ready to go! Thank you

**Comment by jlipps** [comment]
hey, cool to see @karnett ! I just noticed though that `@axe-devtools/axe-appium-xcuitest-driver` is not available on NPM. Are your docs in an accurate state?

**Comment by KazuCocoa** [comment]
Are these drivers available via npm? I still get 404 not found, so we may need to revert these as not available drivers @karnett

**Comment by karnett** [comment]
Hey @jlipps and @KazuCocoa ! sorry for missing these notifications!

Our customers / free trial users need to configure their npm to point to our artifactory where the drivers are published.
It's outlined in the docs (first time setup) but I can put in a request to make it more clear.

If having them available on public npm is a requirement, I can discuss without product manager to see if that's something we want to pursue. Let me know your thoughts - either way I'll still put a request in to clarify the docs. Thanks for the feedback!

**Comment by KazuCocoa** [comment]
@karnett yes, it would be nice to make it available in public in some layers at least.
We haven’t explicitly put requirements but not available packages by default could confuse users. We’d like to keep available drivers/plugins.

**Comment by karnett** [comment]
Thanks for the feedback - this is on our roadmap.

**Comment by KazuCocoa** [comment]
Thank you. Then, could we drop these links until the packages are publicly available from `npm`? @karnett 
 Just we'd like to maintain the page with drivers which are available via `Installation Key` for all

**Comment by karnett** [comment]
Sure @KazuCocoa  - Changes up here: https://github.com/appium/appium/pull/21007

Thanks, we'll be back once we're ready to launch! Also chatting with my team how we can be a better partner going forward :) Thanks for all your work and help here 🖤

**Comment by KazuCocoa** [comment]
thank you! looking forward to being back to the place :)

**Pull request merged**